lang_php/parsing/parser_php.mly

```diff
@@ -1089,6 +1089,8 @@ member_expr:
  | member_expr "->" primary_expr {  ObjGet($1, $2, $3) }
  | member_expr "->" "{" expr "}"
      { ObjGet($1,$2, (BraceIdent ($3, $4, $5))) }
+ | member_expr "->" "..."
```

Comment:
hmm maybe we don't need then the one above with call_expr "->" "..." ?

Reply:
Looks like we still need it. The `dots_method_chaining.php` test in Semgrep fails if I remove it.